core/src/main/java/io/questdb/cutlass/http/MimeTypesCache.java

```diff
@@ -27,9 +27,39 @@
 import io.questdb.std.*;
 import io.questdb.std.str.DirectByteCharSequence;
 import io.questdb.std.str.Path;
+import org.jetbrains.annotations.TestOnly;
+
+import java.io.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public final class MimeTypesCache extends CharSequenceObjHashMap<CharSequence> {
 
+    @TestOnly
```

Comment:
nit: technically this constructor is not test-only as it's used in `DefaultHttpServerConfiguration`

Reply:
damn :) lets remove this later. I really am looking forward to CI completion!